README.md

```diff
@@ -12,14 +12,23 @@ the Jupyter notebook.
   https://github.com/jupyrdf/ipyradiant/workflows/.github/workflows/ciV0.yml/badge.svg
 [status-link]: https://github.com/jupyrdf/ipyradiant/actions
 
-![radiant screencast](https://user-images.githubusercontent.com/32652349/88299868-db232980-ccd0-11ea-9fc8-f21a1047db81.gif)
```

Comment:
we should also be repping `datashader`, `holoviews` and `ipyctoscape`.

Reply:
Do you have a suggestion on the best way to do that?

Reply:
`Powered by [][ipyctoscape], [][datashader], and [][holoviews]`

then have links

Reply:
9fc0a1e